reference/functions/pgroonga-condition.md

```diff
@@ -84,23 +84,228 @@ Please refer to [Calling Functions][sql-syntax-calling-funcs] for information ab
 
 ## Usage
 
-## See also
+### Specify `index_name`
 
-* [postgres_fdw][postgres-fdw]
+シーケンシャルサーチ実行時でも、インデックスに指定したノーマライザーやトークナイザーのオプションを使って検索する方法を紹介します。
 
-* [normalizers_mapping][normalizers-mapping]
+`pgroonga_condition('keyword', index_name => 'pgroonga_index')`を使います。
+`index_name`にノーマライザーやトークナイザーを指定したインデックスの名前を指定します。
+
+Here are sample schema and data:
+
+```sql
+CREATE TABLE tags (
+  name text PRIMARY KEY
+);
+
+CREATE INDEX pgroonga_tag_name_index ON tags
+  USING pgroonga (name pgroonga_text_term_search_ops_v2)
+  WITH (normalizers='NormalizerNFKC150("remove_symbol", true)');
+
+INSERT INTO tags VALUES ('PostgreSQL');
+INSERT INTO tags VALUES ('Groonga');
+INSERT INTO tags VALUES ('PGroonga');
+INSERT INTO tags VALUES ('pglogical');
+```
+
+インデックスサーチ時はインデックスに指定したオプションを使ってインデックスサーチ時の挙動をカスタマイズできます。
+上のサンプルでは、`normalizers='...'`の部分でオプションを指定しています。
+
+一方、インデックスサーチではなくシーケンシャルサーチが実行されると、インデックスに指定されているオプションは参照できません。
+シーケンシャルサーチ時はどのインデックスを参照すればよいかという情報がないからです。
+
+そのためシーケンシャルサーチ実行時は、インデックスサーチ実行時と検索結果が異なる可能性があります。
+この問題を回避するためにシーケンシャルサーチ時に参照するインデックスを明示的に指定します。
+`pgroonga_condition()`の`index_name => '...'`がそのための引数です。
+
```

Comment:
```suggestion
Because of that, there is a possibility that the search results may differ between a sequential search and an index search.
In order to avoid this issue, explicitly specify which index should be referenced at a sequential search.
The `index_name => '...'` argument in `pgroonga_condition()` is used for that.
```